docker/images/localdev-server/workspace/settings.gradle

```diff
@@ -1,9 +1,10 @@
 buildscript {
 	dependencies {
 		classpath group: "biz.aQute.bnd", name: "biz.aQute.bnd", version: "5.2.0"
-		classpath(group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "latest.release") {
+		classpath(group: "com.liferay", name: "com.liferay.gradle.plugins.workspace", version: "5.0.4") {
 			exclude group: "biz.aQute.bnd", module: "biz.aQute.bnd"
 		}
+		classpath group: "org.yaml", name: "snakeyaml", version: "1.32"
```

Comment:
hey Greg, why are you doing this? If you switch to `biz.aQute.bnd.gradle` you won't need this. Also, that fix won't really fix the yaml issue unless you place it in front of bnd.